src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py

```diff
@@ -135,6 +135,8 @@ async def query(self, filters, options):
         options = options or {}
         extra = copy.deepcopy(options.get('extra', {}))
         retrieve_schema = extra.get('include_chart_schema')
+        retrieve_stats = extra.get('stats', False)
+        netdata_metrics = await self.middleware.call('netdata.get_all_metrics') if retrieve_stats else None
```

Comment:
This seems heavy-handed to query all stats on the system just for apps? Can we not query the k3s specific stats explicitly here?

Reply:
@yocalebo i have made changes so that instead of having k8s data split in multiple charts - we gather it in one single chart and query accordingly..